src/actors/actor.go

```diff
@@ -0,0 +1,13 @@
+package actors
+
+import (
+	cid "github.com/ipfs/go-cid"
+)
+
```

Comment:
note: I have moved the methodID's previously defined here to methods.go, any objects?